package.json

```diff
@@ -34,11 +34,11 @@
   },
   "homepage": "https://github.com/hubmapconsortium/vitessce-image-viewer#readme",
   "devDependencies": {
-    "@babel/core": "^7.7.4",
+    "@babel/core": "^7.9.0",
     "@babel/plugin-transform-regenerator": "^7.8.7",
     "@babel/plugin-transform-runtime": "^7.9.0",
-    "@babel/preset-env": "^7.8.3",
-    "@babel/preset-react": "^7.8.3",
+    "@babel/preset-env": "^7.9.0",
+    "@babel/preset-react": "^7.9.0",
```

Comment:
What's up here?  Just want to understand.

Reply:
I get this error when running `npm run start` using the package.json that is currently on the master branch. I think it may be due to the fact that I am on node 14, and it is possibly the same error that @manzt was seeing on travis last week. But upgrading these babel dependencies fixed it.

<img width="945" alt="Screen Shot 2020-05-05 at 9 55 05 AM" src="https://user-images.githubusercontent.com/7525285/81074141-98626a00-8eb6-11ea-8853-f4755ddfe30e.png">

Reply:
However, if the package.json and package-lock changes should be in a different PR I can revert them

Reply:
No it's ok.  Let me check this out and test.